assembly/onpremises-ide-packaging-war-factory/src/main/java/com/codenvy/onpremises/factory/FactoryServlet.java

```diff
@@ -71,11 +71,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         }
 
         // DO NOT REMOVE! This log will be used in statistic analyzing
-        LOG.info("EVENT#factory-accepted# WS#{}# REFERRER#{}# FACTORY#{}# ORG-ID#{}# AFFILIATE-ID#{}# WS-LOCATION#{}# WS-TYPE#{}#",
+        LOG.info("EVENT#factory-accepted# WS#{}# REFERRER#{}# FACTORY#{}# # AFFILIATE-ID#{}# WS-LOCATION#{}# WS-TYPE#{}#",
```

Comment:
I would keep a log entry. This is for now the only way to monitor factories that are being invoked by users and to know from where it comes